inst/examples/joint_rpca_example.qmd

```diff
@@ -0,0 +1,122 @@
+---
+title: "Joint RPCA Reproducible Example"
+format: html
+editor: visual
+---
+
+```{r}
+options(warn = -1)
+# Load user-defined functions
+source("../R/joint_rpca.R")
+source("../R/joint_optspace_helper.R")
+source("../R/joint_optspace_solve.R")
+source("../R/optspace_helper.R")
+source("../R/transform_helper.R")
+source("../R/transform.R")
+source("../R/rclr_transform.R")
+source("../R/mask_value_only.R")
+source("../R/rpca_table_processing.R")
+source("../R/joint_rpca_utils.R")
+
+# TESTING THE JOINT RPCA FUNCTION
```

Comment:
Would it work to demonstrate this with HintikkaXOData? This would save some lines of code as the data is ready (or at least I hope it would be readily usable here).